lineman/app/components/pie_chart/pie_chart.coffee

```diff
@@ -0,0 +1,73 @@
+angular.module('loomioApp').directive 'pieChart', ->
+  template: '<div class="pie-chart"></div>'
+  replace: true
+  scope:
+    votes: '='
+  restrict: 'E'
+  controller: ($scope, $element) ->
+    size = $element.width()
+    paper = new Raphael($element[0], size, size)
+    cx = size / 2.0
+    cy = size / 2.0
```

Comment:
Do we need to use cx and cy or can it just be 'size' everywhere? We don't have a need oblong pie charts I don't think..